### PR TITLE
Fix the subbranch copy/cut paste

### PR DESCRIPTION
**Pull request by EstherDarkish**

### CODE REVIEW

File: src/redux/reducer.ts [hunk 1]
```diff
@@ -950,6 +950,7 @@ export const reducer = createReducer(initialState, (builder) => {
     builder.addCase(NETWORK_MODIFICATION_HANDLE_SUBTREE, (state, action: NetworkModificationHandleSubtreeAction) => {
         if (state.networkModificationTreeModel) {
             let newModel = state.networkModificationTreeModel.newSharedForUpdate();
+
```
Comment (sBouzols) [inline src/redux/reducer.ts:953]: to remove

Comment (EstherDarkish) [inline src/redux/reducer.ts:953]:
```suggestion
            let newModel = state.networkModificationTreeModel.newSharedForUpdate();
```

File: src/redux/reducer.ts [hunk 2]
```diff
@@ -950,6 +950,7 @@ export const reducer = createReducer(initialState, (builder) => {
...
             unravelSubTree(newModel, action.parentNodeId, action.networkModificationTreeNodes);
```
Comment (EstherDarkish) [inline src/redux/reducer.ts:954]:
```suggestion
            let newModel = state.networkModificationTreeModel.newSharedForUpdate();
            unravelSubTree(newModel, action.parentNodeId, action.networkModificationTreeNodes);
```

File: src/redux/reducer.ts (hunk 1 above)
Comment (EstherDarkish) [reply src/redux/reducer.ts]: Ok

**Review by YenguiSeddik — APPROVED** [review]

**Pull request merged**